### PR TITLE
feat(spinners): customize spinners

### PR DESCRIPTION
**Pull request by jean-moldovan**

### CODE REVIEW

File: src/components/ui/icons/Set.vue [hunk 1]
```diff
@@ -43,12 +48,12 @@
 
 <script>
   import Widget from '../../vuestic-components/vuestic-widget/VuesticWidget'
-  import Slider from 'vue-slider-component/src/vue2-slider.vue'
+  import VuesticSlider from 'components/vuestic-components/vuestic-slider/VuesticSlider'
```
Comment (smartapant) [inline src/components/ui/icons/Set.vue]: Add slider to vuestic-components-plugin.js and remove useless import

File: src/components/ui/spinners/Spinners.vue [hunk 2]
```diff
@@ -32,19 +44,25 @@
   import * as spinners from 'epic-spinners'
   import store from 'vuex-store'
   import VuesticColor from 'components/vuestic-components/vuestic-color/VuesticColor'
+  import VuesticSlider from 'components/vuestic-components/vuestic-slider/VuesticSlider'
```
Comment (smartapant) [inline src/components/ui/spinners/Spinners.vue]: Add slider to vuestic-components-plugin.js and remove useless import

File: src/components/ui/spinners/Spinners.vue [hunk 3]
```diff
@@ -32,19 +44,25 @@
   import * as spinners from 'epic-spinners'
   import store from 'vuex-store'
   import VuesticColor from 'components/vuestic-components/vuestic-color/VuesticColor'
```
Comment (smartapant) [inline src/components/ui/spinners/Spinners.vue]: Add color-picker to vuestic-components-plugin.js and remove useless import

File: src/components/vuestic-components/vuestic-color/VuesticColor.vue [hunk 4]
```diff
@@ -10,8 +10,10 @@
       SliderPicker
     },
     props: {
-      value: String,
-      required: true
+      value: {
```
Comment (smartapant) [inline src/components/vuestic-components/vuestic-color/VuesticColor.vue]: Make color picker beautiful - narrow just like slider, play with dot styles

File: src/components/ui/icons/Set.vue (hunk 1 above)
Comment (jean-moldovan) [reply src/components/ui/icons/Set.vue]: Ah, right!
Thanks for pointing this out.

File: src/components/vuestic-components/vuestic-color/VuesticColor.vue (hunk 4 above)
Comment (jean-moldovan) [reply src/components/vuestic-components/vuestic-color/VuesticColor.vue]: ok

File: src/components/ui/spinners/Spinners.vue [hunk 5]
```diff
@@ -62,7 +56,7 @@
           }
         },
         selectedColor: store.getters.palette.primary,
-        selectedSize: 70
+        selectedSize: 60
```
Comment (smartapant) [inline src/components/ui/spinners/Spinners.vue]: try using size modifiers for dotSize/pixelsize/etc properties

File: src/components/ui/spinners/Spinners.vue [hunk 6]
```diff
@@ -43,14 +41,10 @@
...
   import store from 'vuex-store'
-  import VuesticColor from 'components/vuestic-components/vuestic-color/VuesticColor'
-  import VuesticSlider from 'components/vuestic-components/vuestic-slider/VuesticSlider'
 
   export default {
     components: {
-      ...spinners,
-      VuesticColor,
-      VuesticSlider
+      ...spinners
```
Comment (smartapant) [inline src/components/ui/spinners/Spinners.vue]: Please, test ALL spinners for dynamic styles change bugs

**Comment by yevdyko** [comment]
@smartapant After merging #136, I suppose you can close the PR.

**Pull request closed without merging**